packaging/linux/flatpak/dev.lizardbyte.sunshine.yml

```diff
@@ -98,75 +99,56 @@ modules:
-        sha256: d33c56acbbfff2dc540e45c57a38d92210b5e7fd0947ac47fbe48183468aad74
-      - type: shell
-        commands:
-          - for n in $(cat patches/series); do if [[ $n != "#"* ]]; then patch -Np1 -i "patches/$n" -d .; fi; done
+        url: https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz
+        sha256: 06a77bf2ac5c993305882bc1641017f5bec1592d6d1b64787bad492ab34f2f36
+        x-checker-data:
+          type: anitya
+          project-id: 20540
+          stable-only: true
+          url-template: https://www.freedesktop.org/software/libevdev/libevdev-$version.tar.xz
 
-  - name: intel-mediasdk
-    disabled: false
-    buildsystem: cmake
-    config-opts:
-      - -DENABLE_OPENCL=ON
-      - -DENABLE_X11_DRI3=ON
-      - -DENABLE_WAYLAND=ON
-      - -DENABLE_ITT=OFF
-      - -DENABLE_TEXTLOG=OFF
-      - -DENABLE_STAT=OFF
-      - -DBUILD_ALL=OFF
-      - -DBUILD_RUNTIME=ON
-      - -DBUILD_SAMPLES=OFF
-      - -DBUILD_TESTS=OFF
-      - -DBUILD_TOOLS=OFF
-      - -DUSE_SYSTEM_GTEST=OFF
-      - -DMFX_ENABLE_KERNELS=ON
-    only-arches:
-      - x86_64
+  - name: numactl
+    buildsystem: autotools
+    cleanup:
+      - /bin
     sources:
       - type: archive
-        url: http://archive.ubuntu.com/ubuntu/pool/universe/i/intel-mediasdk/intel-mediasdk_22.3.0.orig.tar.gz
-        sha256: e1e74229f409e969b70c2b35b1955068de3d40db85ecc42bd6ff501468bc76d7
-      - type: archive
-        url: http://archive.ubuntu.com/ubuntu/pool/universe/i/intel-mediasdk/intel-mediasdk_22.3.0-1.debian.tar.xz
-        sha256: 024d98d2f63443d2765a90cfe997d104e7b897694889f199ca8fb4d9ffdcf1dc
-      - type: shell
-        commands:
-          - for n in $(cat patches/series); do if [[ $n != "#"* ]]; then patch -Np1 -i "patches/$n" -d .; fi; done
-    modules:
-      - name: libdrm
-        disabled: false
-        buildsystem: meson
-        sources:
-          - type: archive
-            url: http://archive.ubuntu.com/ubuntu/pool/main/libd/libdrm/libdrm_2.4.110.orig.tar.xz
-            sha256: eecee4c4b47ed6d6ce1a9be3d6d92102548ea35e442282216d47d05293cf9737
-          - type: archive
-            url: http://archive.ubuntu.com/ubuntu/pool/main/libd/libdrm/libdrm_2.4.110-1ubuntu1.debian.tar.xz
-            sha256: 464b9553861f39beddfaee6b8924734b02a0febfae3968e4ca1360f2972bba8b
-          - type: shell
-            commands:
-              - for n in $(cat patches/series); do if [[ $n != "#"* ]]; then patch -Np1 -i "patches/$n" -d .; fi; done
+        url: https://github.com/numactl/numactl/archive/refs/tags/v2.0.16.tar.gz
+        sha256: a35c3bdb3efab5c65927e0de5703227760b1101f5e27ab741d8f32b3d5f0a44c
+        x-checker-data:
+          type: anitya
+          project-id: 2507
+          stable-only: true
+          url-template: https://github.com/numactl/numactl/archive/refs/tags/v$version.tar.gz
 
-  - name: numactl
-    buildsystem: autotools
-    make-args:
-      - install
+  - name: xrandr
```

Comment:
This is probably kind of dumb to include just because of the example app that changes resolution. I'd prefer to just remove the "low res desktop" option from the `apps.json`.